Wrappers/Python/cil/optimisation/functions/L1Norm.py

```diff
@@ -236,8 +236,7 @@ def __call__(self, x):
         return y.abs().sum()
 
     def convex_conjugate(self,x):
-        tmp = x.abs().max() - 1
-        if tmp<=1e-8:
+        if x.abs().max() - 1 <=0:
```

Comment:
`x` having value exactly equal to 1 should be allowed:
```suggestion
        if x.abs().max()  - 1 < 0:
```